python/pyspark/pandas/tests/data_type_ops/test_num_ops.py

```diff
@@ -42,7 +54,12 @@ def test_add(self):
                 self.assertRaises(TypeError, lambda: psser + self.non_numeric_pssers["datetime"])
                 self.assertRaises(TypeError, lambda: psser + self.non_numeric_pssers["date"])
                 self.assertRaises(TypeError, lambda: psser + self.non_numeric_pssers["categorical"])
-                self.assertRaises(TypeError, lambda: psser + self.non_numeric_pssers["bool"])
+                self.assert_eq(
+                    (psser + self.non_numeric_pssers["bool"]).sort_index(),
+                    pser + self.non_numeric_psers["bool"],
+                )
+
+            self.assertRaises(TypeError, lambda: self.float_psser + True)
```

Comment:
This is supposed to be out of for-loop?

Reply:
Goot catch! It is out of `with option_context` now.